set-start.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+
+var path = require('path')
+var fs = require('fs')
+
+var save = process.env.npm_config_save
+
+if (save) {
+  var fileName = path.join('..', '..', 'package.json')
+  var file = require(fileName)
+
+  file.scripts = file.scripts || {}
+  file.scripts[ 'start' ] = 'hoodie'
+
+  fs.writeFile(fileName, JSON.stringify(file, null, 2),
+    function (err) {
+      if (err) return console.log(err)
+    })
```

Comment:
two minor things:

1. We try to avoid abbreviations in Hoodie’s code for clarity
2. Can you wrap `return console.log(err)` into `{}` with line breaks?